translation/swift/README.md

```diff
@@ -0,0 +1,75 @@
+# Cloud Translation gRPC Swift Sample
+
+This app demonstrates how to make gRPC connections to the [Cloud Translation API](https://cloud.google.com/translate) to get the translated text.
+
+To call the APIs from iOS, you need to provide authorization tokens with each request for authentication. To get this token, this sample uses a Firebase Function (in Node.js) to generate these tokens on the behalf of a service account.
+
+## Prerequisites
+
+- An OSX machine or emulator
+- [Xcode 10][xcode] or higher
+- [Cocoapods][cocoapods] 
+
+## Setup
+
+- Create a project (or use an existing one) in the [Google Cloud Console][cloud-console]
+- Enable the [Translate API](https://console.cloud.google.com/apis/library/translate.googleapis.com)
+- Enable the [IAM Service Account Credentials API](https://console.cloud.google.com/apis/library/iamcredentials.googleapis.com).
+- [Enable billing][billing]
+- [Create a Service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) with the following IAM roles: `Cloud Translation API Editor` and `Storage Object Admin`. Example name: `translation-glossary`. ([For more info on: how to add roles to a Service Account](https://cloud.google.com/iam/docs/granting-roles-to-service-accounts#granting_access_to_a_service_account_for_a_resource))
```

Comment:
How come `Storage Object Admin`?

Reply:
It's needed for `create glossary` from the application, Since we are not facilitating user to create glossary from the app, so I have removed it and also update the README